bin/setup

```diff
@@ -1,28 +1,18 @@
 #!/usr/bin/env ruby
 require 'pathname'
 
-GEM_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+GEM_ROOT = Pathname.new(__dir__).join("..")
 
 Dir.chdir(GEM_ROOT) do
   if File.exist?("spec/manageiq")
```

Comment:
@bdunne Can you also change this to not do if File.exist? but instead File.file?.  This is because if I have it symlinked I probably *don't* want it to update at all.